core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -60,6 +60,11 @@ import scala.util.control.ControlThrowable
  *            auto.commit.enable=false
  *       3. Mirror Maker Setting:
  *            abort.on.send.failure=true
+ *
+ *      The default partition assignment strategy of the new-consumer-based mirror maker will change from 'range'
```

Comment:
This comment seems like overkill given the message below and the fact that we have a JIRA to track this change.